nemli/commands/utility/summarize.py

```diff
@@ -0,0 +1,41 @@
+import os
+import nextcord
+from openai import OpenAI
+from nextcord.ext import commands
+from nemli.main import bot
+
+# Here we are loading the OpenAI API key from .env and creating a client instance for the OpenAI API
+OPENAI_API_KEY = os.getenv('NEMLI__OPENAI__API__KEY')
```

Comment:
```suggestion
OPENAI_API_KEY = os.getenv('NEMLI__OPENAI__API_KEY')
```

e adicionar isso no `nemli.config`

(pseudocódigo)
```py
class OpenAPI(BaseModel):
    api_key: str
    ...
```

Reply:
Done, depois de uma conversa em privado junto do @ryukinix ✅